@here/harp-mapview/lib/Tile.ts

```diff
@@ -639,6 +646,23 @@ export class Tile implements CachedResource {
         // to be overridden by subclasses
     }
 
+    /**
+     * Update dynamic objects on this tile.
+     */
+    updateDynamicObjects(): void {
+        for (const updater of this.m_dynamicObjectsUpdaters) {
```

Comment:
Can we make this more explicit instead of registering some black box callbacks.
With callbacks it is hard to see how many are registered and what they are doing b/c they are registered in a completely different place.